keyboards/xd75/config.h

```diff
@@ -0,0 +1,145 @@
+/*
```

Comment:
this file should be removed.

Specifically, it resides in `keyboards/xiud/xd75` already.